onegittorulethemall/services/pagure.py

```diff
@@ -174,22 +175,26 @@ def fork_create(self):
     @property
     def fork(self):
         """PagureRepo instance of the fork of this repo."""
-        kwargs = replace_username_with_username(self.pagure_kwargs)
-        kwargs.update(
-            repo=self.repo,
-            namespace=self.namespace,
-            username=self.username,
-            instance_url=self.instance_url,
-            token=self.token,
-            is_fork=True,
-        )
-        fork_project = PagureProject(**kwargs)
-        try:
-            if fork_project.exists and fork_project.pagure.parent:
-                return fork_project
-        except:
-            return None
-        return None
+        if self.forked_project is None:
+            kwargs = replace_username_with_username(self.pagure_kwargs)
```

Comment:
Strange method name.

Reply:
I just moved that code, that's @lachmanfrantisek's invention :dagger: 

I'll correct it.

Reply:
It's now present in #31, I'll rebase this once that one is merged.

Reply:
Oh, I've corrected that as well some time ago, but did not have time to push the changes to usercont/ogr. I'll send a PR so we can use that repo instead of the `onegittorulethemall` dir.